hparams.py

```diff
@@ -251,8 +251,19 @@
     # 0 tends to prevent word repretetion, but sometime causes skip words
     window_backward=1,
     power=1.4,  # Power to raise magnitudes to prior to phase retrieval
+    
+    # GC:
+    # Forced garbage collection probability 
+    # Use only when MemoryError continues in Windows (Disabled by default)
+    #gc_probability = 0.001,
 )
 
+# Preventing Windows specific error such as MemoryError 
+# Also reduces the occurrence of THAllocator.c 0x05 error in Widows build of PyTorch
+if platform.system() == "Windows":
+    print("Windows Detected - num_workers set to 1")
+    hparams.set_hparam('num_workers',1)
+
```

Comment:
Can you move this to `train.py`? Maybe next line of https://github.com/r9y9/deepvoice3_pytorch/blob/52d1026d477c39f2dd12761c461863e732ce86fc/train.py#L878?

Reply:
Okay I will :)